.gitignore

```diff
@@ -2,3 +2,4 @@ build
 phpunit.xml
 composer.lock
 vendor
+.idea
```

Comment:
It is considered best practice to put this in your global git-ignore file. Please remove this.